packages/react-workers/__tests__/sorting.test.js

```diff
@@ -0,0 +1,79 @@
+import { applySorting } from '../src/utils/sorting';
+
+const features = [
+  { column1: 'C', column2: 3 },
+  { column1: 'A', column2: 1 },
+  { column1: 'B', column2: 2 },
+  { column1: 'D', column2: 4 },
+  { column1: 'D', column2: 5 }
+];
+
+const commonSortedFeatures = [
+  { column1: 'A', column2: 1 },
+  { column1: 'B', column2: 2 },
+  { column1: 'C', column2: 3 },
+  { column1: 'D', column2: 4 },
+  { column1: 'D', column2: 5 }
+];
+
+const commonSortedFeatures2 = [
+  { column1: 'D', column2: 4 },
+  { column1: 'D', column2: 5 },
+  { column1: 'C', column2: 3 },
+  { column1: 'B', column2: 2 },
+  { column1: 'A', column2: 1 }
+];
+
+describe('Sorting', () => {
+  describe('should correctly understand sortOptions', () => {
+    test('if undefined', async () => {
+      expect(applySorting(features)).toEqual(features);
+    });
+
+    test('if sortBy is string', async () => {
+      expect(applySorting(features, { sortBy: 'column1' })).toEqual(commonSortedFeatures);
+    });
+
+    test('if sortBy is array of string', async () => {
+      expect(applySorting(features, { sortBy: ['column1', 'column2'] })).toEqual(
+        commonSortedFeatures
+      );
+    });
+
+    test('if sortBy is array of arrays', async () => {
+      expect(applySorting(features, { sortBy: [['column1'], ['column2']] })).toEqual(
```

Comment:
Why is this array of arrays use case required? Could you please share an example or context on it?

Reply:
You might want to order ascendant by column1 and descendant by column 2. You can do that using array of arrays.

Reply:
In the test I just wanted to test the structure, if that works, the different order also works